iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -151,6 +151,7 @@ void PortManager::handlePublisherPorts() noexcept
 void PortManager::doDiscoveryForPublisherPort(PublisherPortRouDiType& publisherPort) noexcept
 {
     publisherPort.tryGetCaProMessage().and_then([this, &publisherPort](auto caproMessage) {
+        cxx::Expects(caproMessage.m_serviceDescription.isValid() && "invalid service description!");
```

Comment:
We definitely want something more modern to print output from Expects. Does not belong in this PR though, I just dislike this style.